meta-mender-extended/recipes-containers/docker/docker-moby_%.bbappend

```
@@ -0,0 +1,31 @@
+python do_prepare_docker_daemon_json() {
```

I would be explicit on the functionality (config file) rather than the format (json)

```suggestion
python do_prepare_docker_daemon_conf() {
```